src/modules/navigator/mission.cpp

```diff
@@ -811,6 +816,11 @@ Mission::read_mission_item(bool onboard, int offset, struct mission_item_s *miss
 		return false;
 	}
 
+	if(offset==0){
```

Comment:
The advance_mission get called too often, i'll see if i can store last notified and move it to set-mission-items